src/common/intl/messages.js

```diff
@@ -30,6 +30,8 @@ var messages = {
   filterOnlyRegular: 'Normal disks',
   filterOnlySnapshots: 'Snapshot disks',
   filterOnlyUnmanaged: 'Unmanaged disks',
+  filterSaveAs: 'Save…',
+  filterSyntaxLinkTooltip: 'To know more about the search syntax, check out the documentation.',
```

Comment:
'Explore the search syntax in our documentation' could be more concise and "sexy". Explore or discover are better (IMHO).